layouts/index.html

```diff
@@ -9,8 +9,8 @@
     </head>
     <body>
         <section id="top" class="section">
-            {{ with .Site.GetPage "/home" }}
             <div class="hero is-{{ .Site.Params.home.introHeight | default "large" }}">
+               {{ with .Site.GetPage "/home" }}
```

Comment:
Why is this change needed?

Reply:
This is due to moving the nav bar inside the section, but the partial won't work if we're in the context of the page so the with needs to move in as well.